### ISSUE TITLE
[bug] Instructions wrong

### ISSUE THREAD

**Issue by MattDHill** [Start9Labs/vaultwarden-wrapper#25]
Instructions seem to indicate that you can use Tor on the iOS Bitwarden app, which is not true. Instead, instructions should have different flows for iOS and Android.

**Comment by BitcoinMechanic** [comment]
Is Tor (via Orbot) expected to work on Android? I have been unable to get it to work.

I get the following error.
![image](https://user-images.githubusercontent.com/45926711/202820345-7666b229-c739-4511-b170-f95734addc71.png)

**Comment by MattDHill** [comment]
Yes, it works for me on my Android. Make sure you have Vaultwarden added as a VPN app and your service URL is reachable from Tor browser.

**Comment by BitcoinMechanic** [comment]
Yes I'm able to reach the service URL via Tor (on firefox).

**Comment by MattDHill** [comment]
So this works, must be something with your setup specifically. Also, why is this on the ticket for updating iOS instructions?

**Comment by BitcoinMechanic** [comment]
It shouldn't be, for a while I thought the ticket needed to be expanded in scope but I don't now.
